exercises/practice/simple-linked-list/example.rs

```diff
@@ -67,10 +67,10 @@ impl<T> FromIterator<T> for SimpleLinkedList<T> {
     }
 }
 
-impl<T> Into<Vec<T>> for SimpleLinkedList<T> {
-    fn into(mut self) -> Vec<T> {
-        let mut vec = Vec::new();
-        while let Some(data) = self.pop() {
+impl<T> From<SimpleLinkedList<T>> for Vec<T> {
+    fn from(mut linked_list: SimpleLinkedList<T>) -> Vec<T> {
+        let mut vec: Vec<T> = vec![];
```

Comment:
```suggestion
        let mut vec: Vec<T> = Vec::with_capacity(linked_list.len());
```

Tiny, unimportant optimization.

Reply:
huh, applying the suggestion failed using "Commit suggestion" and "Add Suggestion to Batch". CLI to the rescue!